Source/WebCore/SourcesCocoa.txt

```diff
@@ -406,6 +406,7 @@ platform/graphics/cocoa/FontPlatformDataCocoa.mm
 platform/graphics/cocoa/GraphicsContextCocoa.mm
 platform/graphics/cocoa/GraphicsContextGLIOSurfaceSwapChain.cpp
 platform/graphics/cocoa/HEVCUtilitiesCocoa.mm
+platform/graphics/cocoa/IconCocoa.mm
```

Comment:
Sort order: this should be after `platform/graphics/cocoa/IOSurfacePoolCocoa.mm`.